packages/ibm-products/src/components/APIKeyModal/APIKeyModal.tsx

```diff
@@ -108,13 +116,24 @@ export let APIKeyModal: React.FC<APIKeyModalProps> = forwardRef(
       ref: copyRef,
     };
     const blockClass = `${pkg.prefix}--apikey-modal`;
+    const localRef = useRef(undefined);
+    const modalRef = (ref || localRef) as MutableRefObject<HTMLDivElement>;
+    const { firstElement, keyDownListener } = useFocus(modalRef);
```

Comment:
Since this is a wrapper around the Carbon modal, we also need to support `selectorPrimaryFocus`, the same way we have done in `TearsheetShell.tsx`.